### PR TITLE
Docs API Update

### PR DESCRIPTION
**Pull request by WillDudley**
- [x] CleanRL tut needs doing
- [x] Ray tut
- [x] Autodoc tutorials
- [x] header_target thing - I'll remove it tomorrow
- [x] Check utils work
- [x] Ideally check which third party envs are refactored, or correctly version controlled
- [x] ray tut docstrings
- [x] update wrappers
- [x] layout check (toc needs fixing, it differs for different pages)

NB: Vectorisation stuff commented out
NB: Depreciated manual pistonball because I don't get it and it doesn't seem to work
NB: Commented out manual control and random demo from api/utils only (need to ideally depreciate everywhere)
NB: Tutorials CI delayed

### CODE REVIEW

**Comment by WillDudley** [comment]
Could someone please make a CleanRL tutorial and link it in /docs/content/tutorials.md?

**Comment by WillDudley** [comment]
@mgoulao What's the header_target thing?

image

**Comment by mgoulao** [comment]
> @mgoulao What's the header_target thing?

I forgot to remove it, sorry. Please remove it :)

**Comment by WillDudley** [comment]
@mgoulao 

Is it possible to autodoc the tutorials (see the structure here: https://github.com/WillDudley/PettingZoo/tree/docs_API_update/tutorials)?

Literally just need a separate page per .py file, showing the code. Also need an appropriate tree structure.

File: docs/api/supersuit_wrappers.md [hunk 1]
```diff
@@ -77,103 +77,172 @@ Supersuit includes the following wrappers:
 
 * `concat_vec_envs_v0(vec_env, num_vec_envs, num_cpus=0, base_class='gym')` takes in an `vec_env` which is vector environment (should not have multithreading enabled). Creates a new vector environment with `num_vec_envs` copies of that vector environment concatenated together and runs them on `num_cpus` cpus as balanced as possible between cpus. `num_cpus=0` or `num_cpus=1` means to create 0 new threads, i.e. run the process in an efficient single threaded manner. A use case for this function is given below. If the base class of the resulting vector environment matters as it does for stable baselines, you can use the `base_class` parameter to switch between `"gym"` base class and `"stable_baselines3"`'s base class. Note that both have identical functionality.
 
-### Parallel Environment Vectorization
+[//]: # (### Parallel Environment Vectorization)
```
Comment (jjshoots) [inline docs/api/supersuit_wrappers.md:80]: This is new to me, what does this do?

Comment (WillDudley) [reply docs/api/supersuit_wrappers.md:80]: Just commenting out markdown (done via pycharm)

Yeah I/we need to review the wrappers part

**Pull request merged**